/var/spool/slurmd/job16830/slurm_script: line 19: nvidia-smi: command not found

[1;34mExtracting features = test/UKB[0m

[1;34mMultivariate analysis for model = test/UKB[0m
Importing outputs/test/UKB/config.cfg...
Traceback (most recent call last):
  File "/home/tbarba/projects/MultiModalBrainSurvival/src/autoencoder/multivariate.py", line 242, in <module>
    merged = create_fulldataset(**config)
  File "/home/tbarba/projects/MultiModalBrainSurvival/src/autoencoder/utils_pred.py", line 29, in create_fulldataset
    features = pd.read_csv(features_path).set_index("eid")
  File "/share/pi/ogevaert/thomas/miniconda/envs/multimodal/lib/python3.9/site-packages/pandas/util/_decorators.py", line 211, in wrapper
    return func(*args, **kwargs)
  File "/share/pi/ogevaert/thomas/miniconda/envs/multimodal/lib/python3.9/site-packages/pandas/util/_decorators.py", line 331, in wrapper
    return func(*args, **kwargs)
  File "/share/pi/ogevaert/thomas/miniconda/envs/multimodal/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 950, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/share/pi/ogevaert/thomas/miniconda/envs/multimodal/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 605, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/share/pi/ogevaert/thomas/miniconda/envs/multimodal/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1442, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/share/pi/ogevaert/thomas/miniconda/envs/multimodal/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1735, in _make_engine
    self.handles = get_handle(
  File "/share/pi/ogevaert/thomas/miniconda/envs/multimodal/lib/python3.9/site-packages/pandas/io/common.py", line 750, in get_handle
    handle = gzip.GzipFile(  # type: ignore[assignment]
  File "/share/pi/ogevaert/thomas/miniconda/envs/multimodal/lib/python3.9/gzip.py", line 173, in __init__
    fileobj = self.myfileobj = builtins.open(filename, mode or 'rb')
FileNotFoundError: [Errno 2] No such file or directory: 'outputs/test/UKB/exports/features/ZZ-UNET_sel2.csv.gz'
